### ISSUE TITLE
Error opening zip file or JAR manifest missing when using qps_client / openloop_client

### ISSUE THREAD

**Issue by normanmaurer** [grpc/grpc-java#2266]
Steps to reproduce:

```
# wget http://repo1.maven.org/maven2/io/grpc/grpc-benchmarks/1.0.0/grpc-benchmarks-1.0.0.tar
# tar xf grpc-benchmarks-1.0.0.tar
# grpc-benchmarks-1.0.0/bin/qps_client
```

```
Error opening zip file or JAR manifest missing : /root/.gradle/caches/modules-2/files-2.1/org.mortbay.jetty.alpn/jetty-alpn-agent/2.0.3/89e34338e9214ca753b33f23ccf80a4b5b861bb7/jetty-alpn-agent-2.0.3.jar
Error occurred during initialization of VM
agent library failed to init: instrument
```

Basically I think you should just comment the `DEFAULT_JVM_OPTS.`

**Comment by ejona86** [comment]
I think this should now be as easy as porting #3037 to benchmarks/.

**Comment by ejona86** [comment]
We should use Conscrypt instead of Jetty ALPN in the benchmarks. We're already using netty_tcnative most of the time, but OkHttp would benefit from Conscrypt.

**Comment by ejona86** [comment]
Fixed in #7368. Jetty ALPN isn't important these days since Java 8 has ALPN support.
